src/dra/dra.h

```diff
@@ -539,12 +539,13 @@ extern Unkstruct_80102CD8 D_801379AC;
 extern s32 D_801379B0;
 extern s32 D_801379B8;
 extern s32 D_801379BC;
+extern SVECTOR D_801379C0;
 extern SVECTOR D_801379C8;
 extern VECTOR D_801379D0;
-extern VECTOR D_801379E0;
-extern VECTOR D_80137B20;
-extern SVECTOR D_80137CA0;
-extern SVECTOR D_80137D40;
+extern VECTOR D_801379E0[];
```

Comment:
```suggestion
extern VECTOR D_801379E0[0x14];
```
0x14 or 20, either of them are fine. It is preferred to put the size of the array in the header so we can both use `LEN(array)` and the PC compiler can validate the array length in the stubs at compilation time